features/system_specs/system_specs.feature

```diff
@@ -1,13 +1,30 @@
 Feature: System spec
 
     System specs are RSpec's wrapper around Rails' own
-    [system tests](http://guides.rubyonrails.org/testing.html#system-testing).
-    We encourage you to familiarize yourself with their documentation.
-
-    RSpec **does not** use your `ApplicationSystemTestCase` helper. Instead it uses
-    the default `driven_by(:selenium)` from Rails. If you want to override this
-    behaviour you can call `driven_by` manually in a test.
-
+    [system tests](https://guides.rubyonrails.org/testing.html#system-testing).
+    
+    > System tests allow you to test user interactions with your application,
+    > running tests in either a real or a headless browser. System tests use
+    > Capybara under the hood.
+    >
+    > By default, system tests are run with the Selenium driver, using the
+    > Chrome browser, and a screen size of 1400x1400. The next section explains
+    > how to change the default settings.
+
+    System specs are marked by `:type => :system` or if you have set
+    `config.infer_spec_type_from_file_location!` by placing them in
+    `spec/system`.
```

Comment:
Can you remove or add a note how this is discouraged? `config.infer_spec_type_from_file_location!` is not recommended.

Reply:
>     System specs are marked by `:type => :system`. If you have set
>     `config.infer_spec_type_from_file_location!` by placing them in
>     `spec/system`. This feature is not recommended, for clarity.
>

Reply:
Oh, that came out garbled, sorry. I'll retry.

Reply:
System specs are marked by `:type => :system`. If you have set
    `config.infer_spec_type_from_file_location!` placing their files in
    `spec/system` also does that. This implicit marking is not recommended, for
    reasons of clarity.

Reply:
Honestly I think I prefer:
> System specs are marked by setting type to :system, e.g. `:type => :system`.
And ignoring the infer command, what that does is documented elsewhere, all its doing after all is adding the type automatically.

Reply:
And it fits on one line!